### ISSUE TITLE
'Enhance' playlist

### ISSUE THREAD

**Issue by apprehensions** [aome510/spotify-player#153]
**Is your feature request related to a problem? Please describe.**
My playlists are empty and i've listened to every song already and i would like to see some new ones.

**Describe the solution you'd like**
Support Spotify's 'Enhance' feature onto playlist actions:
![image](https://user-images.githubusercontent.com/47404953/222890236-51a66583-5716-42f4-86f5-78a35c4de459.png)
Spotify will then add songs, and the user will decide to keep them or not, when kept, Spotify will add more songs.

**Describe alternatives you've considered**
Using the propetiary Spotify client.

**Additional context**
This is only available on the Mobile and Desktop clients; it is not available on the Web, so i'm not sure how this can even BE implemented if at all.

It would be nice though!

**Comment by aome510** [comment]
This is very cool feature in the official app. I personally use it myself a lot on the mobile. Will definitely think about possible ways to implement one. Just a note that Spotify's *cool* features are usually **closed** though xD.

**Comment by ZepsiZola** [comment]
+1 To this. I really like this TUI client, but I also really like how smart-shuffle shows me new songs for the playlist.

**Comment by ZepsiZola** [comment]
I feel like the client could just get songs from the playlist's radio. That's essentially what smart-shuffle is anyway.

**Comment by aome510** [comment]
> I feel like the client could just get songs from the playlist's radio. That's essentially what smart-shuffle is anyway.

yes, but that still requires to implement a custom playback/queueing logic, which can be quite complicated. I have some ideas and will work on a PoC, but there is no concrete timeline yet cause I'm pretty busy nowadays.

**Comment by EnoralTheOutCast** [comment]
Are there any updates on this one? Really love the idea of a TUI, and definitely would love the smart shuffle thing
